osu.Game/Screens/Utility/LatencyCertifierScreen.cs

```diff
@@ -42,12 +45,16 @@ public class LatencyCertifierScreen : OsuScreen
 
         public override float BackgroundParallaxAmount => 0;
 
-        private readonly OsuTextFlowContainer explanatoryText;
+        private readonly LinkFlowContainer explanatoryText;
 
         private readonly Container<LatencyArea> mainArea;
 
         private readonly Container resultsArea;
 
+        public readonly BindableDouble SampleBPM = new BindableDouble(120) { MinValue = 60, MaxValue = 300 };
```

Comment:
```suggestion
        public readonly BindableDouble SampleBPM = new BindableDouble(120) { MinValue = 60, MaxValue = 300, Precision = 0.01 };
```
(or maybe limit to 1, but 0.01 to be consistent with how precise BPMs are)

Reply:
1 was intentional here, which I believe is the default. No one needs to tune their test bpm beyond a unit of 1.

Reply:
Can limit that explicitly then, since right now the slider behaves weirdly without a precision specification (you hear a tick but the value doesn't display as changed, while it's changing in fractions):

https://user-images.githubusercontent.com/22781491/173187745-7fd4f829-f5d9-495d-aa51-fac1a9516286.mp4

```suggestion
        public readonly BindableDouble SampleBPM = new BindableDouble(120) { MinValue = 60, MaxValue = 300, Precision = 1 };
```